R/htmlwidgets.R

```diff
@@ -329,16 +333,18 @@ createWidget <- function(name,
 #' @name htmlwidgets-shiny
 #'
 #' @export
-shinyWidgetOutput <- function(outputId, name, width, height, package = name) {
+shinyWidgetOutput <- function(outputId, name, width, height, inline = FALSE,
+                              package = name) {
```

Comment:
Ack. So it turns out that my own Leaflet package is passing the package argument explicitly, without `package =`, and so breaks with the current htmlwidgets master. @cpsievert, any objection to my putting `inline = FALSE` last?